addon/serializers/firestore.ts

```diff
@@ -33,7 +33,7 @@ export default class FirestoreSerializer extends DS.JSONSerializer {
   }
 
   normalizeCreateRecordResponse(_store: DS.Store, _primaryModelClass: DS.Model, payload: any, id: string | number, _requestType: string) {
```

Comment:
Recommend removing the underscore, since this parameter is now used.
```suggestion
  normalizeCreateRecordResponse(_store: DS.Store, primaryModelClass: DS.Model, payload: any, id: string | number, _requestType: string) {
```